apps/web/components/Charts/v2/constants.ts

```diff
@@ -0,0 +1,65 @@
+import { theme } from '@island.is/island-ui/theme'
+
+import { ChartComponentType, FillPattern } from './types'
+
+/**
+ * A list with pixels that define how the line should be displayed
+ * 4 4 => 4 pixels filled 4 pixels empty
+ * 4 2 => 4 pixels filled 2 pixels empty
+ * 4 1 1 1 => 4 pixels filled 1 pixel empty 1 pixel filled 1 pixel empty
+ * ... and so on
+ */
+export const PREDEFINED_LINE_DASH_PATTERNS = ['16 2', '16 2 4 2', '8 2', '3 2']
+
+/**
+ * Pattern implementations are found in generateFillPattern() from utils.tsx
+ */
+export const PREDEFINED_FILL_PATTERNS: FillPattern[] = [
+  FillPattern.diagonalSeToNw,
+  FillPattern.diagonalSwToNe,
+  FillPattern.diagonalSeToNwDense,
+  FillPattern.diagonalSwToNeDense,
+  FillPattern.dotsSmall,
+  FillPattern.dotsMedium,
+  FillPattern.dotsLarge,
+  FillPattern.horizontal,
+  FillPattern.vertical,
+]
+export const PREDEFINED_PIE_FILL_PATTERNS: FillPattern[] = [
+  FillPattern.dotsSmall,
+  FillPattern.diagonalSeToNw,
+  FillPattern.dotsMedium,
+  FillPattern.diagonalSwToNe,
+  FillPattern.dotsLarge,
+  FillPattern.diagonalSeToNwDense,
+  FillPattern.diagonalSwToNeDense,
+]
+
+/**
+ * By default this will be the order of colors given to chart components
+ * Unless they specifically opt to overwrite them
+ */
+export const DEFAULT_COLORS = [
+  theme.color.blue300,
+  theme.color.blueberry300,
+  theme.color.purple300,
+  theme.color.roseTinted300,
+  // '#0061FF',
```

Comment:
Are these colors here commented out for a reason, I'm mostly wondering if we can delete the comments if we won't use the hex colors here